playbooks/roles/hammer_devel/tasks/hammer_config.yml

```diff
@@ -19,6 +19,20 @@
         :username: {{ hammer_devel_username }}
         :password: {{ hammer_devel_password }}
 
+- name: 'Configure hammer-cli-foreman-admin'
+  blockinfile:
+    dest: ~/.hammer/cli.modules.d/foreman_admin.yml
+    create: true
+    block: |
+      :foreman_admin:
+        :enable_module: true
```

Comment:
Yeah, we can use lineinfile to update config settings to be appropriate for this box. I'll get on it.